<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Watch</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-----QMenuBar-----*/
QMenuBar { background-color: #F8F8F8; color: #000000; border-color: #000000; }
QMenuBar::disabled { background-color: #404040; color: #656565; border-color: #051a39; }
QMenuBar::item { background-color: transparent; }
QMenuBar::item:selected { background-color: #9F2425; color: #ffffff; }
QMenuBar::item:pressed { background-color: #9F2425; border: 1px solid #000; margin-bottom: -1px; padding-bottom: 1px; }

/*-----QMenu-----*/
QMenu { background-color: #F8F8F8; border: 1px solid; color: #000; }
QMenu QWidget { background-color: transparent; }
QMenu::separator { height: 1px; background-color: #6d8eff; color: #ffffff; padding-left: 4px; margin-left: 10px;  margin-right: 5px; }
QMenu::item { min-width : 150px; padding: 3px 20px 3px 20px; }
QMenu::item:selected { background-color: #9F2425;  color: #f0f5f3;}
QMenu::item:disabled { color: #262626; }

/*-----QSlider-----*/
QSlider::sub-page:Horizontal { background-color: #9F2425; }
QSlider::add-page:Horizontal { background-color: #333333; }
QSlider::groove:Horizontal { background: transparent; height:4px; }
QSlider::handle:Horizontal { width:10px; border-radius:5px; background:#9F2425; margin: -5px 0px -5px 0px; }

/*-----QLineEdit-----*/
QLineEdit::focus { border: 1px solid #9F2425; border-radius: 3px; }

/*-----QPushButton-----*/
QPushButton { border: 1px solid #D0D0D0; border-radius: 10px; background-color: #FEFEFE;}
QPushButton::pressed { background-color: #D0D0D0; border-color: #D0D0D0}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="Player" name="player" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_local"/>
    <addaction name="actionOpen_url"/>
    <addaction name="actionShare_current"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionFullscreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="DiscreetDock" name="networkDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Network</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="networkDockContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="NetworkDisplay" name="networkDisplay" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen_local">
   <property name="text">
    <string>Open local file...</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionOpen_url">
   <property name="text">
    <string>Open url...</string>
   </property>
  </action>
  <action name="actionShare_current">
   <property name="text">
    <string>Share current media</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Player</class>
   <extends>QWidget</extends>
   <header>HMI/Player/player.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NetworkDisplay</class>
   <extends>QWidget</extends>
   <header>HMI/networkdisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DiscreetDock</class>
   <extends>QDockWidget</extends>
   <header>HMI/Convenience/discreetdock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
